package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "prebuild": "dotenv -- cross-env yarn supabase:generate:remote",
-    "dev": "wrangler dev --env dev --port 4000",
+    "dev": "wrangler dev --env dev --port 4003",
```

Comment:
```suggestion
    "dev": "wrangler dev --env dev --port 4000",
```

Reply:
I think it would be better to have them selecting a higher port automatically so we don't have to change the local configuration all the time but that can be changed later.